java/client/src/main/java/glide/api/models/commands/json/JsonGetOptionsBinary.java

```diff
@@ -0,0 +1,67 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.json;
+
+import static glide.api.models.GlideString.gs;
+
+import glide.api.commands.servermodules.Json;
+import glide.api.models.GlideString;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Builder;
+
+/** GlideString version of additional parameters for {@link Json#get} command. */
+@Builder
+public final class JsonGetOptionsBinary {
```

Comment:
I think you can have single options class with extended builder to accept strings and glidestrings both

Reply:
Again, lots of other Option classes do this. Maybe not the best design. Let's leave it for future refactoring.